Ryujinx/Ui/MainWindow.cs

```diff
@@ -963,6 +964,11 @@ private void CreateGameWindow()
 
             DisplaySleep.Prevent();
 
+            //if (runOnce)
+            //{
+                RendererWidget.ComboPressed += OnComboPressed;
+            //}
+
```

Comment:
```suggestion

            RendererWidget.ComboPressed += OnComboPressed;

```
Properly align and remove the commented code.